trieste/acquisition/function/function.py

```diff
@@ -325,6 +463,63 @@ def __call__(self, x: TensorType) -> TensorType:
         return ei * augmentation
 
 
+class log_augmented_expected_improvement(AcquisitionFunctionClass):
```

Comment:
here you can inherit from `augmented_expected_improvement`...

Reply:
Fixed in https://github.com/secondmind-labs/trieste/pull/904/commits/2f45eb8e8035fbfa7105f6f2acd9a470edb32906